bundles/org.openhab.core/src/main/java/org/openhab/core/library/unit/Units.java

```diff
@@ -319,6 +319,10 @@ public final class Units extends CustomUnits {
         SimpleUnitFormat.getInstance().alias(MONTH, "mo");
         SimpleUnitFormat.getInstance().alias(WEEK, "wk");
         SimpleUnitFormat.getInstance().alias(YEAR, "y");
+
+        // other aliases
+        SimpleUnitFormat.getInstance().alias(MIRED, "mired");
+        SimpleUnitFormat.getInstance().alias(MIRED, "mirek");
```

Comment:
These could probably be moved up right after line 296? All the others have their alias right after the label, except for the few works-arounds just above.

Reply:
Ok. Done.